### PR TITLE
Added ingress host with no proxy only for demo environment

### PR DESCRIPTION
**Pull request by nitinprabhu**

### CODE REVIEW

**Review by linusnorton — DISMISSED** [review]

**Review by damongreen123 — APPROVED** [review]

File: k8s/namespaces/nfdiv/nfdiv-frontend/demo.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  releaseName: nfdiv-frontend
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/nfdiv-frontend
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      ingressClass: traefik-no-proxy
```
Comment (timja) [inline k8s/namespaces/nfdiv/nfdiv-frontend/demo.yaml:15]: do you need the replicas interpod and ingress host all set here?

Comment (nitinprabhu) [reply k8s/namespaces/nfdiv/nfdiv-frontend/demo.yaml:13]: removed good point

**Review by timja — APPROVED** [review]

File: k8s/namespaces/nfdiv/nfdiv-frontend/demo.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-frontend
+spec:
+  releaseName: nfdiv-frontend
+  chart:
```
Comment (timja) [inline k8s/namespaces/nfdiv/nfdiv-frontend/demo.yaml:7]: oops, also should remove this, saw just as i clicked approve

Comment (nitinprabhu) [reply k8s/namespaces/nfdiv/nfdiv-frontend/demo.yaml:7]: will remove that :)

**Pull request merged**